### ISSUE TITLE
Fix Windows support

### ISSUE THREAD

**Issue by anthonymonori** [dipien/bye-bye-jetifier#28]
**Describe the bug**
Using byeByeJetifier 1.1.0 with default configuration, [the exclusion rule in the default configuration]( https://github.com/dipien/bye-bye-jetifier/blob/v1.1.0/bye-bye-jetifier/src/main/java/com/dipien/byebyejetifier/ByeByeJetifierExtension.kt#L24-L25) provided with the version doesn't seem to be picked up when running byeByeJetifier

**To Reproduce**
1. Add `implementation 'com.facebook.android:facebook-android-sdk:9.0.0'` as a dependency
2. Run `gradle :canISayByeByeJetifier -Pandroid.enableJetifier=false`
3. See False positive:
```
 Issues found:
 * com\facebook\appevents\codeless\internal\ViewHierarchy.class -> android.support.v4.view.NestedScrollingChild
```

**Expected behavior**
Above issue not showing

**Comment by maxirosson** [comment]
Thanks for reporting this issue. Are you using Windows?

**Comment by anthonymonori** [comment]
Hey @maxirosson , yes I am. I haven't considered that to be the cause, but jow that you say I was indeed running this on Windows.

**Comment by maxirosson** [comment]
I pushed a possible fix to master, could you please try It on Windows before releasing a new version?

You need to put the following configuration to use the snapshot:

```
buildscript {
	repositories {
		maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
	}
	dependencies {
		classpath("com.dipien:bye-bye-jetifier:1.1.1-SNAPSHOT")
	}
}
```

**Comment by maxirosson** [comment]
@anthonymonori If you can, please send me the task report using v1.1.1-SNAPSHOT. Thanks

**Comment by anthonymonori** [comment]
So weird stuff, but it feels like none of the exceptions are being picked up? I'm getting more issues when I run it with `1.1.1` and no other dependency change

```
=========================================
Project: app
=========================================

Scanning com.android.installreferrer:installreferrer:1.0
 Absoulute path: C:\Users\Antal\.gradle\caches\modules-2\files-2.1\com.android.installreferrer\installreferrer\1.0\7f4644e967eec4ad13290da5806ad1d4d6698994\installreferrer-1.0.aar
 Graphs to this dependency:
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-core:9.0.0
          +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-common:9.0.0
          +---com.facebook.android:facebook-core:9.0.0
               +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-login:9.0.0
          +---com.facebook.android:facebook-common:9.0.0
               +---com.facebook.android:facebook-core:9.0.0
                    +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-share:9.0.0
          +---com.facebook.android:facebook-common:9.0.0
               +---com.facebook.android:facebook-core:9.0.0
                    +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-gamingservices:9.0.0
          +---com.facebook.android:facebook-share:9.0.0
               +---com.facebook.android:facebook-common:9.0.0
                    +---com.facebook.android:facebook-core:9.0.0
                         +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-gamingservices:9.0.0
          +---com.facebook.android:facebook-common:9.0.0
               +---com.facebook.android:facebook-core:9.0.0
                    +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-login:9.0.0
          +---com.facebook.android:facebook-core:9.0.0
               +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-share:9.0.0
          +---com.facebook.android:facebook-core:9.0.0
               +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-gamingservices:9.0.0
          +---com.facebook.android:facebook-share:9.0.0
               +---com.facebook.android:facebook-core:9.0.0
                    +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-applinks:9.0.0
          +---com.facebook.android:facebook-core:9.0.0
               +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-messenger:9.0.0
          +---com.facebook.android:facebook-core:9.0.0
               +---com.android.installreferrer:installreferrer:1.0
 +---com.facebook.android:facebook-android-sdk:9.0.0
     +---com.facebook.android:facebook-gamingservices:9.0.0
          +---com.facebook.android:facebook-core:9.0.0
               +---com.android.installreferrer:installreferrer:1.0
 Issues found:
 * com\android\installreferrer\api\InstallReferrerClient$Builder.class -> android/support/annotation/UiThread
 * com\android\installreferrer\api\InstallReferrerClient.class -> android/support/annotation/UiThread
 * com\android\installreferrer\api\InstallReferrerClient.class -> android/support/annotation/NonNull
 * com\android\installreferrer\api\InstallReferrerClientImpl$InstallReferrerServiceConnection.class -> android/support/annotation/NonNull
 * com\android\installreferrer\api\InstallReferrerClientImpl.class -> android/support/annotation/NonNull
```

Plus several others that suddenly popped up but would be too long of a stacktrace to copy:
```
Scanning org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.0
 Absoulute path: C:\Users\Antal\.gradle\caches\modules-2\files-2.1\org.jetbrains.kotlinx\kotlinx-coroutines-android\1.3.0\804a29e0b22cfb30e04487574ad1cb888daa716d\kotlinx-coroutines-android-1.3.0.jar
 Graphs to this dependency:
 +---androidx.room:room-ktx:2.2.6
     +---org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.0
 Issues found:
 * kotlinx\coroutines\android\HandlerDispatcherKt.class -> android/support/annotation/VisibleForTesting
 * kotlinx\coroutines\android\AndroidExceptionPreHandler.class -> android/support/annotation/Keep

**Comment by maxirosson** [comment]
@anthonymonori I pushed another fix to master. Could you please run the following command:

`
./gradlew canISayByeByeJetifier --refresh-dependencies
`

and attach me a txt with the report? So I can review it and see if a find any issue on Windows.

Remember to use the following configuration:

```
buildscript {
	repositories {
		maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
	}
	dependencies {
		classpath("com.dipien:bye-bye-jetifier:1.1.1-SNAPSHOT")
	}
}
```

**Comment by anthonymonori** [comment]
So, I have the following setup:
- Snapshots repository added as described above
- Using 1.1.1-SNAPSHOT version
- `android.enableJetifier=true` set in gradle.properties

Running the following command:
`./gradlew :canISayByeByeJetifier -Pandroid.enableJetifier=false --refresh-dependencies`

Produces the following output:
[output-111-with-refresh-dependencies.txt](https://github.com/dipien/bye-bye-jetifier/files/5975668/output-111-with-refresh-dependencies.txt)

Same command, but using 1.1.0 produces the following output:
[output-110-with-refresh-dependencies.txt](https://github.com/dipien/bye-bye-jetifier/files/5975670/output-110-with-refresh-dependencies.txt)

**Comment by maxirosson** [comment]
@anthonymonori Sorry, could you please try again? The snapshot you used wasn't properly generated. I fixed the snapshot, so you can try again now. Thanks.

**Comment by anthonymonori** [comment]
Sure! This time it worked as expected:
[output-111-snapshot.txt.txt](https://github.com/dipien/bye-bye-jetifier/files/5976654/output-111-snapshot.txt.txt)

**Comment by maxirosson** [comment]
Great. Thanks for your help !!!

**Comment by maxirosson** [comment]
v1.1.1 release with this fix.

**Comment by maxirosson** [comment]
Released v1.1.2 with the fix
